doc/CMakeLists.txt

```diff
@@ -213,4 +213,11 @@ if (BUILD_DOCS)
                ${CMAKE_BINARY_DIR}/swig/python/osgeo/osr.py
              WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
 
+    # Check docstub annotations
+    add_test(NAME docstub-annotations
```

Comment:
This command is actually generating the annotations, right? Would it make sense to register a custom target for the stub files, rather than going through `ctest` ? CI would still fail if the target cannot be built.